src/redux/store.js

```diff
@@ -0,0 +1,7 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+const store = configureStore({
+  reducer: {},
+});
```

Comment:
- Kindly, import and add books and categories to configure the store as it is part off the requirements 👌👌
```
const store = configureStore({
  reducer: { books: booksReducer, categories: categoriesReducer },
})

```

Reply:
Thanks for sharing this example, I will implement it in my project